Users/handlers.py

```diff
@@ -0,0 +1,106 @@
+import re
+import hashlib
+import uuid
+from datetime import datetime
+import tornado.web
+import tornado.ioloop
+import tornado.httpserver
+import tornado.options
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+from .models import User
+from DatabaseConnect.DbConnection import DbConnection
+
+
+class UserHandler(DbConnection, tornado.web.RequestHandler):
+    '''
+    Request Handler for users table
+    Methods used=('GET ,'POST','DELETE')
+    '''
+
+    def validate_email(self, email):
+        # Validating the format of email
+        if not email:
+            self.set_status(400)
+            self.finish(dict(error=True, message="Email is missing"))
+        if re.match("[^@]+@[^@]+\.[^@]+", email) != None:
```

Comment:
This regex works for properly formatted emails but fails for **abcd@pqrs@gmail.com** by accepting it as valid.

Reply:
Okay

Reply:
Done